PLIPs/plip-1.md

```diff
@@ -0,0 +1,218 @@
+* **Draft** -- Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the PLIP to be mature and ready to proceed to the next status. A PLIP in draft status must be implemented to be considered for promotion to the next status (ignore this requirement for core PLIPs).
+  * :arrow_right: Last Call -- If agreeable, the PLIP editor will assign Last Call status and set a review end date (`review-period-end`), normally 14 days later.
+  * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that PLIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
+* **Last Call** -- This PLIP will soon be listed prominently on the https://plips.plasma.group/ website once we get it set up.
+  * :x: -- A Last Call which results in material changes or substantial unaddressed technical complaints will cause the PLIP to revert to Draft.
+  * :arrow_right: Accepted (Core PLIPs only) -- A successful Last Call without material changes or unaddressed technical complaints will become Accepted.
+  * :arrow_right: Final (Not core PLIPs) -- A successful Last Call without material changes or unaddressed technical complaints will become Final.
+* **Accepted (Core PLIPs only)** -- This PLIP is in the hands of the Plasma client developers.  Their process for deciding whether to encode it into their clients is not part of the PLIP process.
+  * :arrow_right: Final -- Standards Track Core PLIPs must be implemented in multiple viable Plasma clients before it can be considered Final. When the implementation is complete and adopted by the community, the status will be changed to “Final”.
+* **Final** -- This PLIP represents the current state-of-the-art. A Final PLIP should only be updated to correct errata.
+
+Other exceptional statuses include:
+
+* **Deferred** -- This is for core PLIPs that have been put off for a future release.
+* **Rejected** -- A PLIP that is fundamentally broken or a Core PLIP that was rejected by the Core Devs and will not be implemented.
+* **Active** -- This is similar to Final, but denotes a PLIP which may be updated without changing its PLIP number.
+* **Superseded** -- A PLIP which was previously final but is no longer considered state-of-the-art. Another PLIP will be in Final status and reference the Superseded PLIP.
+
+## What belongs in a successful PLIP?
+
+Each PLIP should have the following parts:
+
+- Preamble - RFC 822 style headers containing metadata about the PLIP, including the PLIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](#plip-header-preamble) for details.
+- Simple Summary - “If you can’t explain it simply, you don’t understand it well enough.” Provide a simplified and layman-accessible explanation of the PLIP.
```

Comment:
IMO add Terminology as well. A lot of jargon in our community:) Good idea for a first PLIP that standardizes teminology.